src/order/upper_lower/basic.lean

```diff
@@ -266,6 +267,31 @@ alias is_lower_set_iff_Iio_subset ↔ is_lower_set.Iio_subset _
 
 end partial_order
 
+section linear_order
+variables [linear_order α] {s t : set α}
+
+lemma is_lower_set.total (hs : is_lower_set s) (ht : is_lower_set t) : s ⊆ t ∨ t ⊆ s :=
```

Comment:
This is a duplicate, right?

Reply:
Yeah sorry I accidentally committed my working file.